test/helpers/cilium.go

```diff
@@ -336,7 +336,11 @@ func (s *SSHMeta) GetEndpointsNames() ([]string, error) {
 // containing policies, DaemonSets, etc.) are stored for the runtime tests.
 // TODO: this can just be a constant; there's no need to have a function.
 func (s *SSHMeta) ManifestsPath() string {
-	return fmt.Sprintf("%s/runtime/manifests/", BasePath)
+	return fmt.Sprintf("%s/runtime/manifests/", s.basePath)
```

Comment:
The commit message for `[CI] Move BaseString to SSHMeta and Kubectl structs` is wrong, since it's BasePath. Also, since it's a more general change, could you add some explanation in the commit ?